NAAncyra-1/index.html

```diff
@@ -26,15 +26,22 @@
     <body>
         <nav class="navbar--content">
             <img class="navbar--logo" src="images/logo.png" alt="" />
-            <h3>Embrace Spirit of the Capital</h3>
-            <h3>3-4-5 NOVEMBER</h3>
+            <div class="navbar--text navbar--slogan">
+                <span class="fw-lighter">Embrace Spirit of</span>
+                <span class="fw-bold"> the Capital</span>
+            </div>
+            <div class="navbar--text navbar--date">
+                <span class="">3-4-5 </span>
+                <span class="fw-bold">NOVEMBER</span>
+            </div>
         </nav>
 
         <div class="about container">
             <div class="about--content">
                 <img src="images/about.jpg" alt="" />
-                <div class="about--text">
-                    <h3 class="fw-bold">About NA Ancyra '23</h3>
+                <div class="about--text mb-">

```

Comment:
şu başlığı diğer sectionlar gibi içeriğin üstüne (fotoğraf dahil) taşısak daha iyi olur gibi, bi de eğer burada About'u incelttiysek diğer yerlerde (Reach, Organising gibi) de inceltelim, standart olsun